develop/tutorials/articles/110-portlets/15-javascript/04-npm/08-npm-sdk.markdown

```diff
@@ -1,171 +1,208 @@
+number which is compatible according to semantic versioning, you are still in 
+the same feature level (or maybe a higher one if you increase version numbers 
+enough to reach a new level). This can lead to running an unsupported npm SDK 
+version that may be broken or produce unexpected results. For example, say that 
+feature level 2 is the maximum possible, and your component versions are 
+slightly higher than the minimum versions required (Loader modules extender has 
+increased from version 1.1.0 to 1.5.0):
+
+- Gradle: 2.5.0
+- Gradle Node plugin: 2.3.0
+- npm: 4.0.0
+- Babel: 6.26.0
+- Liferay npm bundler: 1.2.1
+- Loader modules extender: 1.5.0
+- AMD loader: 2.1.0
+
+You are still in feature level 2, as all version changes have been semantical 
+versioning compatible. If, however, you update to these version numbers:
+
+- Gradle: 2.5.0
+- Gradle Node plugin: 2.3.0
+- npm: 4.0.0
+- Babel: 6.26.0
+- Liferay npm bundler: 1.5.2
+- Loader modules extender: 2.1.0
+- AMD loader: 2.1.0
+
+You are no longer in feature level 2, as you have introduced the possibility of 
+a breaking change when updating Loader Modules Extender from 1.1.0 to 2.1.0. In 
+this case, you would be in an indeterminate feature level (pre-3 for example) 
+that has not been tested. We recommend that you use component versions that are 
+semantical version compatible with the component versions listed in the feature 
+level you wish to use. 
+
++$$$
+
+<!-- `liferay-npm-build-tools` version only shows up in your level 8 example in the README. Running 
+the tool does not list this component. Is it suppose to be listed, or has this 
+changed?  -->
+
+**Note:** The liferay-npm-build-tools version specifies the version for the 
+liferay-npm-bundler tool, the plugins for Babel, and the plugins for 
+liferay-npm-bundler. All these components are always released together to make 
+version management easier. That means that some subcomponent may have different 
+version numbers for the same artifact, but we prefer such redundancy to having 
+disparate version numbers for each subcomponent.
+
+$$$
+
+## Using the npm SDK Assistant Tool [](id=using-npm-sdk-assistant-tool)
+
+Follow these steps to use the npm SDK Assistant:
+
+1.  Run `npm install -g liferay-npm-sdk-assistant` to install the npm SDK 
+    assistant tool. 
+    
+2.  Navigate to your project's root folder and run `lnka man features` to list 
+    the supported feature levels.
+    
+3.  Run `lnka features` to list your current feature level.
+
+4.  Update any components that don't satisfy the minimum version requirements 
+    for the feature level you want.
+
+The available commands are listed below.
+
+### Available Commands [](id=available-commands)
+    
+This tool supports the commands listed below:
+
+- `lnka help`: Lists all of the supported commands <!-- appears to do the same 
+as `lnka man` -->
+
+- `lnka <command> help`: Obtains help about a specific command
```

Comment:
@izaera 
I was unable to run all the options shown for `lnka features help`. how do you run them? With a sever running, `lnka features --server` for instance produces an error. How do you configure this?